classes/core.php

```diff
@@ -4672,7 +4672,7 @@ static public function render_form($atts, $entry_id = null, $shortcode = null)
 				// retain query string
 				$qurystr = array();
 				parse_str($_SERVER['QUERY_STRING'], $qurystr);
-				$out .= "<span class=\"caldera-grid\"><ol class=\"breadcrumb\" data-form=\"caldera_form_" . $current_form_count . "\" id=\"caldera-forms-breadcrumb_" . $current_form_count . "\">\r\n";
+				$out .= "<div class=\"caldera-grid\"><ol class=\"breadcrumb\" data-form=\"caldera_form_" . $current_form_count . "\" id=\"caldera-forms-breadcrumb_" . $current_form_count . "\">\r\n";
```

Comment:
@iamntz In isolation, you are correct. This is more semantic.

I am fairly opposed to this change because a change in element type can effect CSS targeting and JavaScript bindings. How series of a concern is that? How many of the 100-200k sites running CF does that matter to? No idea.

But I could be convinced that I'm over-reacting.